src/triage/cli.py

```diff
@@ -168,6 +175,12 @@ def __init__(self, parser):
             default=1,
             help="number of cores to use",
         )
+        parser.add_argument(
+            "--matrix-format",
+            choices=self.matrix_storage_map.keys(),
+            default="csv",
+            help="The matrix storage format to use"
```

Comment:
would be nice to include the choices and default in the description – since otherwise it's opaque – _e.g._:

```suggestion
            help="The matrix storage format to use (either {choices}, default {default})".format(
                choices=' or '.join(self.matrix_storage_map),
                default=next(iter(self.matrix_storage_map)),
            ),
```

Reply:
I tested it out and argcmdr was helpful enough to include the choices by default. What if we just added the default to the description?

Reply:
I think having the default be `next(iter())` is a bit of a misdirection, and would rely on it being an OrderedDict and that nobody messed with the order ever.

If we want to avoid having the string 'csv' in two places that are not right next to each other maybe right under the `matrix_storage_map` definition I can put a `matrix_storage_default` that is csv? The advantage is this would be slightly easier to keep updated if we ever changed the choice names.

Reply:
1. Yeah I guess `argparse` includes the choices in the signature? But not in the `help`? (Certainly less important if it's included *somewhere*.)

2. Python dictionaries now guarantee insertion order, so it seemed like a reasonable balance between verbosity and DRY. I don't think it's a disaster if someone messes with that dictionary's order -- that just changes the default. A comment could explain it.

    Or of course something else would do, (though likely with a bit more verbosity).

Reply:
I also got it in the help.

```
  --matrix-format {csv,hdf}
                        The matrix storage format to use. [default: csv]
```

Reply:
Well, at least I think that's close enough to the help to work